include/gfxbase.h

```diff
@@ -262,12 +273,12 @@ class GFXBase : public Adafruit_GFX
 
     virtual void drawPixel(int16_t x, int16_t y, CRGB color)
     {
-        addColor(xy(x, y), color);
+        leds[xy(x, y)] = color;
     }
 
     virtual void drawPixel(int16_t x, int16_t y, uint16_t color)
     {
-        addColor(xy(x, y), from16Bit(color));
+        leds[xy(x, y)] = from16Bit(color);
```

Comment:
Not sure why I'd chosen to do additive color here, other than it looked cool in the Ghostwave effect.  But it needs to be strict pixel replacement, otherwise drawing black doesn't work